PYME/Acquire/Hardware/Piezos/piezo_c867.py

```diff
@@ -310,16 +334,18 @@ def close(self):
             #time.sleep(.01)
             #self.ser_port.close()            
                 
-        
-    def SetServo(self, state=1):
-        self.lock.acquire()
-        try:
-            self.ser_port.write(b'SVO 1 %d\n' % state)
-            self.ser_port.write(b'SVO 2 %d\n' % state)
-            self.servo = state == 1
-        finally:
-            self.lock.release()
-            
+    # directly enabling/disabling the servo outside the thread loop
+    #      seems to hang PYME
+    # the implementation below instead 'messages' the thread loop by setting flags
+    #      that get picked up in thread loop processing
+    # I dislike state being an int and rather have opted treating as bool which reflects the on/off state better IMHO
+    def SetServo(self, state=True):
```

Comment:
We would need to check with other piezos / stages that might use more than 1 state. 

Although a boolean might ultimately make sense, the `SetServo` method is part of more general piezo/stage interface, so we can't change it arbitrarily here - we would need to check all `SetServo` usages across other piezos / stages and all calling functions.